flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/excamation/ExclamationWithStormBolt.java

```diff
@@ -86,11 +99,43 @@ public static void main(final String[] args) throws Exception {
 	// USER FUNCTIONS
 	// *************************************************************************
 
-	private static class ExclamationMap implements MapFunction<String, String> {
+	private static class ExclamationMap extends AbstractRichFunction implements MapFunction<String, String> {
+
+		private String exclamation;
+
+		@Override
+		public void open(Configuration parameters) throws Exception {
+			RuntimeContext ctx = super.getRuntimeContext();
```

Comment:
You should not use `JobConfiguration` for regular Flink functions. Instantiate `exlamation` via constructor Argument `exlcamationNum`.